crates/mpz-ole-core/src/ideal/mod.rs

```diff
@@ -0,0 +1,7 @@
+//! This module provides ideal functionalities for OLE and ROLE.
```

Comment:
```suggestion
//! Ideal OLE functionalities.
```

They're all just variants of OLE, we can keep the module doc general